libecho/src/Echo.cc

```diff
@@ -8,10 +8,11 @@
 #include <QCoreApplication>
 #include <QDebug>
 
-static constexpr int windowSize = 200;
-static constexpr int loFreq = 19000;
-static constexpr int hiFreq = 20000;
-static constexpr double magLimit = 300.0 / windowSize;
+static constexpr int windowSize = 300;
+static constexpr int loFreq = 14500;
+static constexpr int hiFreq = 15000;
```

Comment:
Is it necessary to use those frequencies? One of project's basic ideas is to use high frequencies that human can barely hear.

Reply:
testowałem na moim kompie różne; 19k i 20k, jakie były poprzednio, tworzyły błędy, zaś to dawało dobre wyniki